sensor_msgs/Image image
---
BoundingBox[] faces